src/backend/oracle-data-interface/src/main/java/ca/bc/gov/trafficcourtsonline/oracledatainterface/model/Dispute.java

```diff
@@ -0,0 +1,65 @@
+package ca.bc.gov.trafficcourtsonline.oracledatainterface.model;
+
+import java.sql.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+//mark class as an Entity   
+@Entity  
```

Comment:
You've included Lombok, but not used yet.  If you want, you can remove all the getters and setters in this file and replace with these annotations.
Or remove Lombok from the pom.xml
```suggestion
@Entity  
@Getter @Setter @NoArgsConstructor
```

Reply:
I've never used Lombok before but saw it in another project and wanted to try. I didn't get a chance to use and discover other features of Lombok yet but this looks like a good start.

Reply:
Ah, k. I've never used it before either, but have seen it in a template on another project. I'm not convinced I like it yet, but am totally willing to give it a try.